moss_test.go

```diff
@@ -1,51 +1,65 @@
 package gomoss
 
-import "testing"
+import (
+	"bytes"
+	"os"
+	"testing"
+)
 
 func TestMossInit(t *testing.T) {
-    mossID := uint32(12345678)
+	mossID := uint32(12345678)
 	request := NewRequest(mossID)
 
-    switch {
-    case request.ID != mossID:
-        t.Fail()
-    case request.IgnoreLimit != 10:
-        t.Fail()
-    case request.ExperimentalServer != 0:
-        t.Fail()
-    case request.MatchingNumber != 250:
-        t.Fail()
-    case request.DirectoryMode != 0:
-        t.Fail()
-    case request.Language != "c":
-        t.Fail()
-    }
+	switch {
+	case request.ID != mossID:
```

Comment:
nit: use the idiomatic [golang way of testing](https://www.golang-book.com/books/intro/12):

```go
if request.ID != mossID {
  t.Errorf("Expected request.ID to be %q, got %q", request.ID, mossID)
} 
```

that way we can see all failures for a run instead of just getting a failure with no output.